src/components/file/file.tsx

```diff
@@ -133,17 +133,13 @@ export class File {
     private chipSet;
     private mdcTextField;
 
-    public connectedCallback() {
-        this.initialize();
-    }
```

Comment:
I realised my comment shows on the wrong code in the Conversation tab, so I'll add it here as well 🙂

> You can't just remove the connectedCallback. This will break the component if it's ever detached from the DOM and then re-attached. Like if a consumer wants to move the component from one place in the DOM to another.